workspaces/adr/plugins/adr-backend/src/index.ts

```diff
@@ -20,6 +20,5 @@
  * @packageDocumentation
  */
 
-export * from './search';
```

Comment:
I think the types haven't been removed. Is it possible to export all the types as before, mark them as deprecated so that we don't mark this change as breaking? It will make life easier for adopters

Reply:
Hi @vinzscam - sorry to push back a bit but my understanding is that a deprecation period is only required for packages that are already on major versions (https://backstage.io/docs/overview/versioning-policy/#package-versioning-policy)

From what I've seen historically we've always regularly released 0.x.0 breaking minor changes for pre-major version packages to allow for more freedom to evolve the package rapidly until it reaches a stable v1 release

Reply:
What should I do here then? I've addressed the other change, but I prefer to wait until I know if I should do it a `patch` or `minor` change, to avoid going back and forth with this little piece

Reply:
I'll defer to @backstage/community-plugins-maintainers here but my preference as the plugin owner is to ship as is as a breaking change for the rationale laid out in my previous comment

Reply:
Alright. So if they prefer here to go with the patch+deprecation, I'll bring that back again and follow that process instead. I'll wait a little before doing that anyway, just in case